great_expectations/datasource/fluent/fluent_base_model.py

```diff
@@ -39,6 +42,50 @@
 _Self = TypeVar("_Self", bound="FluentBaseModel")
 
 
+def _exclude_name_fields_from_fluent_datasources(
+    config: Dict[str, Any]
+) -> Dict[str, Any]:
+    if "fluent_datasources" in config:
+        fluent_datasources: dict = config["fluent_datasources"]
```

Comment:
Why is this in the `FluentBaseModel`?
I think this only applies when `GxConfig` model tries to serialize.

Reply:
@Kilo59 I put these utilities here, because they are used only in this module.